devTools/Dockerfile-php80-xdebug

```diff
@@ -25,8 +20,16 @@ RUN set -eux; \
     apk add --no-cache --virtual .phpexts-rundeps ${runDeps}; \
     apk del .build-deps
 
+RUN apk add --no-cache \
+        ncurses \
+        make \
+        bash \
+        expect \
+        zip
+
 COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
 COPY devTools/memory-limit.ini ${PHP_INI_DIR}/conf.d/memory-limit.ini
+COPY devTools/xdebug-coverage.ini ${PHP_INI_DIR}/conf.d/xdebug-coverage.ini
```

Comment:
this can be a bit optimized to decrease amount of useless layers:
`COPY devTools/memory-limit.ini devTools/xdebug-coverage.ini ${PHP_INI_DIR}/conf.d/`

Reply:
Sure, done.